benefits/static/css/styles.css

```diff
@@ -177,7 +177,7 @@ footer {
   font-weight: 500;
   margin-bottom: 1rem;
   padding: 1rem;
-  width: 100%;
+  width: 49%;
```

Comment:
This is affecting other pages. I _think_ putting `width: 100%` back and moving `width: 49%` into your style class should solve it, but please check it.

Comparison of the agency-index page on your branch vs. the `dev` branch:
![image](https://user-images.githubusercontent.com/25497886/163283578-1480c542-f78a-48e1-b058-fd21810af5eb.png)

![image](https://user-images.githubusercontent.com/25497886/163283595-7a1d9bb0-212d-42f2-80e7-23f934aa1bcf.png)